addons/crm/models/res_config_settings.py

```diff
@@ -15,6 +15,10 @@ class ResConfigSettings(models.TransientModel):
     group_use_recurring_revenues = fields.Boolean(string="Recurring Revenues", implied_group='crm.group_use_recurring_revenues')
     # Membership
     is_membership_multi = fields.Boolean(string='Multi Teams', config_parameter='sales_team.membership_multi')
+    module_membership = fields.Boolean("Membership")
+    membership_type = fields.Selection(
```

Comment:
This should not be defined in CRM, as it has no usage. Don't see how crm should be impacted by this.

Reply:
Are you introducing a configuration parameter to know if we name partners, Partner or Members ? I would say this is quite weird, especially you can have both, partners (customers) and some members.

If we remove the membership module (which is the case), don't talk about members anymore. But you can make a "Members" menu in "membership", using the right label. That's generally how we do things in Odoo, modules have their own labelling :) .